public/js/action-list.js

```diff
@@ -116,15 +148,12 @@
                     : activeItems[activeItems.length - 1].dataset.icingaDetailFilter;
             }
 
-            if (isBeingMultiSelected && container.id === 'col1' && list.matches('.content > :scope')) {
-                // The restriction on col1 is currently only in place as it's not defined what should happen
-                // with primary lists in col2. (as a detail load then moves the list to col1)
-                container.dataset.suspendAutorefresh = '';
+            if (! dashboard) {
+                _this.addSelectionCountToFooter(list);
```

Comment:
This must be called after `clearSelection()` and `setActive()`. Otherwise the counter in the footer is not up to date.